src/org/usfirst/frc/team166/robot/commands/ElevatorOn.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc.team166.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.usfirst.frc.team166.robot.Robot;
+
+/**
+ *
+ */
+public class ElevatorOn extends Command {
+
+	public ElevatorOn() {
+		requires(Robot.elevator);
+	}
+
+	// Called just before this Command runs the first time
+	@Override
+	protected void initialize() {
+		Robot.elevator.start();
+	}
+
+	// Called repeatedly when this Command is scheduled to run
+	@Override
+	protected void execute() {
+
+	}
+
+	// Make this return true when this Command no longer needs to run execute()
+	@Override
+	protected boolean isFinished() {
+		return false;
```

Comment:
Is this command going to be assigned to a button? If not, this needs to know when to finish somehow.